SPRINT-STRUCTURE.md

```diff
@@ -1,115 +1,73 @@
-      - What should be in my standup update
-    - Recipes
-      - How to run efficient standups for remote teams
-- Day 3
-  - [Pull Requests](code-reviews/pull-requests.md) (separate from code reviews)
-    - Conceptual requirements for pull request (it should build, have 1 reviewer, linked work item, build changes)
-      - Add emphasis on importance of protecting master, effect this has on crew efficiency
-    - Recipe for Setup in
-      - Azure DevOps
-      - GitHub
-    - [Code Reviews](code-reviews/README.md)
-      - Conceptual
-        - Add to checklist (breaking changes & backward compatibility, security, fault tolerance, etc)
-  - Code Merging
-    - prescribe strategy (i.e. squash /w or w/o rebase)
-- Day 4
-  - [Continuous Deployment](continuous-deployment/readme.md) (extend, much more explanation needed)
-    - Conceptual, Purpose, Goals, Impact and Measures
-      - Which environments (ci, test, stage)? For each environment...
-        - Conceptually whats is the purpose for each env
-        - When should deployment trigger
-        - Pre-deployment approvers
-        - Sign off for promotion
-    - Recipes for Setting up CD Pipelines
-      - Azure DevOps
-  - Asserting Test Cases and Automation
-- Day 5
-  - Sprint Demo
-  - [Retrospectives](retrospectives/readme.md)
-    - Conceptual
-      - Inputs (Requirements to have ready before meeting)
-      - Participants required
-      - Outputs (Decisions, actions to conclude meeting)
-    - Guide for retrospective facilitator
-      - Timeline for 1 hour retro
-      - Tips for sticking to time
-      - Voting for action items
-    - Recipes
-      - Remote retros using ADO Retrospectives
-      - Remote retros using Retrium
-  - [Grooming](backlog-management/grooming/readme.md)
-    - Conceptual
-      - Inputs (Requirements to have ready before meeting)
-      - Participants required
-      - Outputs (Decisions, actions to conclude meeting)
-    - Definition of ready for stories
-      - Examples of well defined acceptance criteria
-      - Can the story be tested as written
-      - Can it be completed within a sprint
-      - Is it dependent on other stories
-    - [Estimation](sprint-planning/estimation/readme.md)
-      - Resolving estimation conflicts (two people are sizing differently))
+### Day 1
 
-## Assumptions
+- [ ] [Plan the first sprint](sprint-planning/readme.md)
+  - Agree on a sprint goal, and how to measure the sprint progress
+  - Determine team capacity
+  - Assign user stories to the sprint and split user stories into tasks
+  - Set up Work in Progress (WIP) limits
+- [ ] [Decide on test frameworks and discuss test strategies](test-first-development/readme.md)
+  - Discuss the purpose and goals of tests and how to measure test coverage
+  - Agree on how to separate unit tests from integration, load and smoke tests
+- [ ] [Decide on branch naming for feature branches](source-control/feature-branching/readme.md)
```

Comment:
Maybe we can make this more generic by removing "for feature branches"? There are other areas you may want to define (bugs for example)